### PR TITLE
Create Employee text resolved as success. Config.cs connection string…

### PR DESCRIPTION
**Pull request by Figamus**
## Link to Ticket
Closes # [2](https://github.com/NSS-Therapeutic-Raccoons/WorkforceManagement/issues/2)

## Description of Proposed Changes
- Add Employee
- Resolved add employee test as success.

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout df-employee-add
Run Post_CreateAddsEmployee Test
Run BangazonWorkforce
Select Employee
Enter Fields
Create New Employee
```

## Impacted Areas in Application

List general components of the application that this PR will affect:

* Employee Create New.

## Mentions @username

@khardt
@figamus
@jeremiah3643
@ricky-bruner
@askingalot

## Definition of Done

1. Can manually create a new Employee.
2. Can run Post_CreateAddsEmployee test.

### CODE REVIEW

**Review by ricky-bruner — APPROVED** [review]
When testing, remember to add your connection string to the Config.cs file or the test will fail.

All tests work great, as well as the functionality!

File: BangazonWorkforce/BangazonWorkforce.IntegrationTests/EmployeeTests.cs [hunk 1]
```diff
@@ -51,7 +51,7 @@ public async Task Post_CreateAddsEmployee()
 
             string newFirstName = "FirstName-" + Guid.NewGuid().ToString();
             string newLastName = "LastName-" + Guid.NewGuid().ToString();
-            string isSupervisor = "true";
```
Comment (askingalot) [inline BangazonWorkforce/BangazonWorkforce.IntegrationTests/EmployeeTests.cs]: Why did you change `isSupervisor` from `true` to `false`?

**Review by askingalot — CHANGES_REQUESTED** [review]
Please resolve these items:

* I get the following exception when I try to add a new employee
```
SqlException: The INSERT statement conflicted with the FOREIGN KEY constraint "FK_EmployeeDepartment". The conflict occurred in database "Raccoons", table "dbo.Department", column 'Id'.
The statement has been terminated.
```
* add Config.cs to .gitignore
* expand the "steps to test" section of the PR description
* add comments to EmployeeController

File: BangazonWorkforce/BangazonWorkforce.IntegrationTests/EmployeeTests.cs (hunk 1 above)
Comment (Figamus) [reply BangazonWorkforce/BangazonWorkforce.IntegrationTests/EmployeeTests.cs]: I was testing to see if it worked with false. Just never changed it back

**Comment by ricky-bruner** [comment]
Andy, I believe you may need to run the SQL query because I am not getting any sql exceptions on manual or integrated testing. Your database tables may no longer match ours.

**Review by askingalot — CHANGES_REQUESTED** [review]
Please expand the "steps to test" section of the PR description.

**Review by askingalot — APPROVED** [review]
